src/serverless.js

```diff
@@ -168,6 +168,14 @@ class GraphQL extends Component {
     // deploy role
     const { roleArn } = await aws.utils.deployRole(deployRoleParams)
 
+    // Add policy to the Lambda role to allow VPC CreateNetworkInterface
+    await new aws.IAM()
+      .attachRolePolicy({
+        RoleName: this.state.name,
+        PolicyArn: 'arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole'
```

Comment:
Added ```{
  Effect: 'Allow',
  Action: [
    'logs:CreateLogGroup',
    'logs:CreateLogStream',
    'logs:PutLogEvents',
    'ec2:CreateNetworkInterface',
    'ec2:DescribeNetworkInterfaces',
    'ec2:DeleteNetworkInterface'
  ],
  Resource: '*'
}``` to the inline policy. It would be nice, for this circumstance, if there was a way reference a aws managed policy in the inline policy, but that would be strange and it doesn't seem like there's a way.